### PR TITLE
Removing OCM tool from rhmi-utils repo

### PR DESCRIPTION
**Pull request by STEPHINRACHEL**
**Motivation**
https://issues.redhat.com/browse/DEL-203
**What**
Removing OCM tool from rhmi-utils repo since the tool has moved to delorean repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,54 +174,3 @@ $ zcat ~/repos/rhmi-utils/samples/xyz.json.gz | jq -f scripts/resources_by_names
 "openshift-webapp",0.002,87613440,,,,,
 "ops-health-monitoring",,,,,,,
 ```
-
-## Using `ocm` for installation of RHMI
```
Comment (psturc) [inline README.md:178]: I would keep this heading and add a note that the ocm tooling was moved to delorean, wdyt

**Review by psturc — CHANGES_REQUESTED** [review]
Just one comment and also as we discussed we should include the deletion of [measure-downtime.js](https://github.com/integr8ly/rhmi-utils/blob/master/scripts/ocm/measure-downtime.js) script and move it to delorean repo as well and make sure it is merged today, so the nightly pipeline won't fail.

File: README.md (hunk 1 above)
Comment (STEPHINRACHEL) [reply README.md:178]: Yes, that make sense

**Pull request closed without merging**